### ISSUE TITLE
Confused by the B_avg

### ISSUE THREAD

**Issue by AnthonyF333** [4uiiurz1/pytorch-adacos#1]
Hello, I want to know B_avg = torch.where(one_hot < 1, self.s * torch.exp(logits), torch.zeros_like(logits)) is right?
According to the original paper, I think B_avg = torch.where(one_hot < 1, torch.exp(self.s * logits), torch.zeros_like(logits))

**Comment by 4uiiurz1** [comment]
You are right!
Thanks for letting me know.
